src/com/lftechnology/java/training/dipak/employeemanagement/controller/EmployeeController.java

```diff
@@ -0,0 +1,216 @@
+                    role = UserType.ADMIN;
+                } else if (role2 == 2) {
+                    role = UserType.USER;
+                }
+                break;
+            } catch (Exception ex) {
+                LOGGER.log(Level.INFO, exceptionOccurred, ex);
+                LOGGER.info("Please re-enter the values.");
+            }
+
+        }
+
+        employee.setAddress(address);
+        employee.setDepartment(department);
+        employee.setFullName(fullName);
+        employee.setIsTerminated(isTerminated);
+        employee.setUserName(userName);
+        employee.setPassword(password);
+        employee.setRole(role);
+
+        EmployeeService ed = ServiceFactory.getEmployeeService();
+        count = ed.addEmployee(employee);
+
+        return count;
+    }
+
+    /**
+     * <p>
+     * This method takes the name of the employee to be terminated and call terminateEmployee method in service layer. This method returns
+     * the number of employees terminated.
+     * </p>
+     * 
+     * @param e
+     * @param sc
+     * @return count
+     */
+    public int terminateEmployee(Employee em, Scanner sc) {
+
+        String fullName = null;
+        int count = 0;
+        try {
+            LOGGER.info("Please enter the employee fullname::");
+            fullName = sc.nextLine();
+            em.setFullName(fullName);
+
+            EmployeeService ed = ServiceFactory.getEmployeeService();
+
+            ed.terminateEmployee(em);
+
+        } catch (Exception ex) {
+            LOGGER.log(Level.INFO, exceptionOccurred, ex);
+        }
+        return count;
+    }
+
+    /**
+     * <p>
+     * This method takes the input for the filters and calls the viewEmployee method.
+     * </p>
+     * 
+     * @param e
+     * @param sc
+     * @return
+     */
+    public ResultSet viewEmployee(Employee empl, Scanner sc) {
+
+        String fullName = null;
+        String department = null;
+        String address = null;
+        try {
+            LOGGER.info("Please enter the employee fullname::");
+            fullName = sc.nextLine();
+
+            LOGGER.info("Please enter the department::");
+            department = sc.nextLine();
+
+            LOGGER.info("Please enter the address::");
+            address = sc.nextLine();
+
+            empl.setFullName(fullName);
+            empl.setAddress(address);
+            empl.setDepartment(department);
+
+            EmployeeService ed = ServiceFactory.getEmployeeService();
+
+            ed.viewEmployee(empl);
+
+        } catch (Exception ex) {
+            LOGGER.log(Level.INFO, exceptionOccurred, ex);
+        }
+        return null;
+    }
+
+    /**
+     * <p>
+     * This method takes the input for the fields to be edited and calls editEmployeeDetails in service layer.
+     * </p>
+     * 
+     * @param e
+     * @param sc
+     * @return e
+     */
+    public Employee editEmployeeDetails(Employee emp1, Scanner sc) {
+        Employee e = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "e" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)